client/src/pages/flights/index.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import '../../App.css';
+import './flight.css';
+import Header from '../../components/header';
+import Sidenav from '../../components/sidenav';
+import Table from '../../components/table';
+
+class Flight extends Component {
+  render() {
+    return (
+        <div className="flight-container" >
+          <Header />
```

Comment:
Because the 3 admin pages (flights, add-flight, update-flight) share a lot of layout (header, sidenav), another component structure you could try is to have one top-level Admin component which has a Header, Sidenav, and then swaps between the 3 admin pages based on the user's actions.  Something like:
```
class AdminDashboard extends Component {
render() {
    return (
         <div className='admin-dashboard'>
          <Header />
         ...
          <Sidenav>
          {this.props.selectedAdminPage === 'updateFlight' && (<UpdateFlightPage />)}
          {this.props.selectedAdminPage === 'addFlight' && (<AddFlightPage />)}
          ...

```
If you were going to use react-router, it could have a `<Switch>` and `<Route>` statements inside there too.

Reply:
Using react-router is in our plan :smiley: